### PR TITLE
Integrate Mayhem

### PR DESCRIPTION
**Pull request by rnshah9**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM --platform=linux/amd64 ubuntu:20.04
+
+RUN apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y build-essential gcc-multilib
+
```
Comment (xansec) [inline Dockerfile:5]: Please use stages.

File: Makefile [hunk 2]
```diff
@@ -21,7 +21,7 @@ CONFIG ?= release
 
 CC ?= gcc
 CFLAGS ?= -std=c11
-CFLAGS += -Werror -Wall -Wextra -Wvla -Wstrict-aliasing -Wstrict-overflow=5 -Wshadow -Wstrict-prototypes -Wmissing-prototypes -Wmissing-declarations -Wmissing-field-initializers -g
+CFLAGS += -Wall -Wextra -Wvla -Wstrict-aliasing -Wstrict-overflow=5 -Wshadow -Wstrict-prototypes -Wmissing-prototypes -Wmissing-declarations -Wmissing-field-initializers -g
```
Comment (xansec) [inline Makefile:24]: Can you comment on this change?

File: Mayhemfile [hunk 3]
```diff
@@ -0,0 +1,6 @@
+project: rnshah9/fcc
```
Comment (xansec) [inline Mayhemfile]: Please use generic project names and correct filepath directives.

**Review by xansec — CHANGES_REQUESTED** [review]

**Comment by rnshah9** [comment]
Removing `-Werror` fixes the build by allowing compiler warnings.

**Comment by xansec** [comment]
Makes sense to me!

**Pull request merged**